roles/publisher/files/environment.yml

```diff
@@ -0,0 +1,7 @@
+name: esgf-pub
+channels:
+  - defaults
```

Comment:
Should we add the esgf channel?

Reply:
If it has the packages (cmor and dependencies) that we need. We will then need to copy future releases as we update versions.

Reply:
Gotcha.  I still have some things to address that we discussed last week (whether or not to include the build numbers).  So I'll go ahead and approve and merge.  We can add the esgf channel later.